src/components/FilterSearch.tsx

```diff
@@ -91,7 +104,6 @@ export function FilterSearch({
   }, [filterSearchResponse?.sections]);
 
   const hasResults = sections.flatMap(s => s.results).length > 0;
-  const [currentFilter, setCurrentFilter] = useState<Filter>();
 
   const handleDropdownEvent = useCallback((itemData, select) => {
     const newFilter = itemData?.filter as Filter;
```

Comment:
do we also need to set the query when arrowing down onto an option or clicking on an option? I'm imagining a couple scenarios: 
- user types the first couple characters of input and then clicks an option without arrowing down (e.g. type "t" and clicks on "Tom Meyer"). I don't think the query would be updated to the latter
- where the user types some input text and selects a filter, then clicks back into the input and arrows down onto a different dropdown option, but then clicks out of filter search. basically the behavior when swapping between two filters, one is unselected and then the other is selected, so it might clear out the query text but not replace it with new text

Reply:
updated to handleSelectDropdown and handleToggleDropdown to update top level `query` variable with `setQuery` using the new value param from those two events.